### ISSUE TITLE
Annotation of "y compris"

### ISSUE THREAD

**Issue by bguil** [surfacesyntacticud/guidelines#51]
The annotation of "y compris" is inconsistent in French treebanks:

 - [in GSD](https://universal.grew.fr/?custom=656f51d06e1f5) and [in Rhapsodie](https://universal.grew.fr/?custom=656f51d06e1f5&corpus=SUD_French-Rhapsodie%40latest): `PRON` + `VERB` and `ExtPos=ADP`
 - [in Sequoia](https://universal.grew.fr/?custom=656f52de323fa): `PRON` + `ADJ` with various `ExtPos`
 - [in FQB](https://universal.grew.fr/?custom=656f5380b7c58): `PRON` + `VERB` and `ExtPos=CCONJ`
 - [in ParTUT](https://universal.grew.fr/?custom=656f5380b7c58&corpus=UD_French-FQB%402.13): `PRON` + `VERB` 

I propose to make all annotations following GSD and Rhapsodie.

**Comment by simeng-song** [comment]
The token “compris” has already been annotated as upos=VERB, ExtPos=CCONJ, and lemma="comprendre" in the French treebanks.

As for its syntactic dependencies, a comp:obl relation is currently annotated between “y” and “compris” in the GSD. Since “compris” is now treated as a CCONJ, should we add a cc dependency pointing to it, as well as a conj:coord between the coordinated elements before and after “y compris”?

**Comment by sylvainkahane** [comment]
Yes cc and conj:cordEnvoyé de mon smartphoneLe 20 juin 2025 à 16:59, simeng-song ***@***.***> a écrit :﻿simeng-song left a comment (surfacesyntacticud/guidelines#51)
The token “compris” has already been annotated as upos=VERB, ExtPos=CCONJ, and lemma="comprendre" in the French treebanks.
As for its syntactic dependencies, a comp:obl relation is currently annotated between “y” and “compris” in the GSD. Since “compris” is now treated as a CCONJ, should we add a cc dependency pointing to it, as well as a conj:coord between the coordinated elements before and after “y compris”?
